### PR TITLE
Brazilian Portuguese Translation

### PR DESCRIPTION
**Pull request by kaudy**
I'm working on the translation for Brazilian Portuguese. 

-The Translation is currently 55%
-All Translations with "*" must be re-calibrated.

Estou trabalhando na tradução para o português brasileiro. Eu criei esse tópico para discutirmos sobre os termos usados na tradução.

-A tradução está atualmente em 55%
-Todas as traduções com "*" precisam ser novamente verificadas.

### CODE REVIEW

**Comment by Gymnasiast** [comment]
You'll have to adjust your changes to language.c and language.h since Italian has been added in the meantime.

**Comment by kaudy** [comment]
ok, tkss @Gymnasiast, I will add the files.
the translation is not yet complete due to some terms that can not fit the Portuguese but could merge the project so that subsequent updates are made to the repository of translations.

**Comment by runelaenen** [comment]
Language changes should be done in the other language repo
https://github.com/OpenRCT2/Localisation

**Comment by Gymnasiast** [comment]
@runelaenen: Not when you're _adding_ an extra language. See the changes in language.c (not present in the Localisation repo.)

**Comment by kaudy** [comment]
@runelaenen if you think best, I create a new pull to the portugues_br.txt language file in languages repository and create a new pull to update the language.c and language.h the develop repository

File: projects/openrct2.sln [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
+# Visual Studio Express 2013 for Windows Desktop
```
Comment (marcotc) [inline projects/openrct2.sln]: There seems to be no need for this change.

**Comment by Gymnasiast** [comment]
The support code has been implemented in #1416, the translation itself has now been moved to https://github.com/OpenRCT2/Localisation/blob/master/data/language/portuguese_br.txt

**Comment by kaudy** [comment]
tks @IntelOrca and @Gymnasiast

**Pull request closed without merging**